.rubocop.yml

```diff
@@ -12,8 +12,7 @@ AllCops:
     - 'tmp/**/*'
     - 'spec/smoke_tests/**/*.rb'
 
-# See https://github.com/rubocop-hq/rubocop/issues/6410
```

Comment:
Do you mind to keep this reference? I believe it's here to explain why this is disabled.